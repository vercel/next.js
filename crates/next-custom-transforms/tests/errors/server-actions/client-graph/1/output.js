/* __next_internal_client_entry_do_not_use__ default auto */ export default function App() {
    async function fn() {
        'use server';
    }
    return <div>App</div>;
}
