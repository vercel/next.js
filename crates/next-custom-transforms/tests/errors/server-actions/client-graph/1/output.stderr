  x It is not allowed to define inline "use server" annotated Server Actions in Client Components.
  | To use Server Actions in a Client Component, you can either export them from a separate file with "use server" at the top, or pass them down through props from a Server Component.
  | 
  | Read more: https://nextjs.org/docs/app/api-reference/functions/server-actions#with-client-components
  | 
   ,-[input.js:5:1]
 4 |   async function fn() {
 5 |     'use server'
   :     ^^^^^^^^^^^^
 6 |   }
   `----
